frontend/packages/rhoas-plugin/locales/ja/rhoas-plugin.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Do we need to provide translations? How to deal with those? Leave those empty?

Reply:
Do not create the other locales, they get created later.

Reply:
+1